Legion/VaultOfTheWardens/Locales/koKR.lua

```diff
@@ -18,15 +18,15 @@ end
 
 L = BigWigs:NewBossLocale("Tirathon Saltheril", "koKR")
 if L then
-	--L.warmup_trigger = "I will serve MY people, the exiled and the reviled."
+	L.warmup_trigger = "나는 추방당하고 버려진 "나의" 일족을 섬기겠다."
```

Comment:
You need to escape those quotes with a backslash, like this: `L.warmup_trigger = "나는 추방당하고 버려진 \"나의\" 일족을 섬기겠다."`

Reply:
No, no. The string itself must be inside normal quotes: `string = "this is a string"`, they indicate the start and end of it. Basically, right now, your string ends with "버려진 ", then Lua sees 2 symbols it can't recognize, then it sees yet another string: "일족을 섬기겠다" . If you want to make a string that contains quotes, you need to add a backslash before those that are a part of the string: `\"`, as in: `string = "this is a string with a \"word\" surrounded by quotes"`.

Reply:
ah ok. I'll add the backslashes when I get home from work.